### PR TITLE
remove optional usage from protobuf messages, remove unused RenameDatabaseObjectRequest and RenameDatabaseObjectResponse

### PR DESCRIPTION
**Pull request by pjain1**

### CODE REVIEW

**Review by begelundmuller — CHANGES_REQUESTED** [review]

File: proto/rill/runtime/v1/api.proto [hunk 1]
```diff
@@ -1008,23 +1007,20 @@ message GetCardinalityOfColumnRequest {
 message GenerateTimeSeriesRequest {
   string instance_id = 1;
   string table_name = 2;
-  optional BasicMeasures measures = 3;
+  repeated BasicMeasureDefinition basic_measures = 3;
```
[the remark this answers is not in the document]
Comment (pjain1) [reply proto/rill/runtime/v1/api.proto]: changed

File: runtime/server/column_api.go [hunk 2]
```diff
@@ -46,10 +47,16 @@ func (s *Server) GetTopK(ctx context.Context, topKRequest *runtimev1.GetTopKRequ
 	}
 	for rows.Next() {
 		var topKEntry runtimev1.TopK_TopKEntry
-		err = rows.Scan(&topKEntry.Value, &topKEntry.Count)
+		var value interface{}
```
[the remark this answers is not in the document]
Comment (pjain1) [reply runtime/server/column_api.go]: changed

File: runtime/server/column_api.go [hunk 3]
```diff
@@ -46,10 +47,16 @@ func (s *Server) GetTopK(ctx context.Context, topKRequest *runtimev1.GetTopKRequ
...
+		err = rows.Scan(&value, &topKEntry.Count)
 		if err != nil {
 			return nil, status.Error(codes.Internal, err.Error())
 		}
+		if value == nil {
+			topKEntry.Value = structpb.NewNullValue()
+		} else {
+			topKEntry.Value = structpb.NewStringValue(value.(string))
```
[the remark this answers is not in the document]
Comment (pjain1) [reply runtime/server/column_api.go]: changed

**Review by begelundmuller — APPROVED** [review]

**Comment by begelundmuller** [comment]
> Please re-generate the frontend bindings (`npm run generate:runtime-client -w web-common`) and wait for CI to pass, then we can merge this

Actually, I just did it. Will merge when CI passes

**Pull request merged**